### PR TITLE
data.atm: finish converting authors to authors@R, add a few namespaces

### PR DESCRIPTION
**Pull request by infotroph**
## Description

Figured I'd sweep some dust bunnies out of the corners as long as I was looking at check results.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
-  [x] cleanup
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/data.atmosphere/DESCRIPTION [hunk 1]
```diff
@@ -3,13 +3,12 @@ Type: Package
...
-Authors@R: c(person("Mike","Dietze"),
-             person("David","LeBauer"),
-             person("Carl", "Davidson"),
-             person("Rob","Kooper"),
-             person("Deepak", "Jaiswal"))
-Author: Carl Davidson, Mike Dietze, Rob Kooper, Deepak Jaiswal, David LeBauer
-Maintainer: David LeBauer <dlebauer@illinois.edu>
+Authors@R: c(person("Mike","Dietze", role = "aut"),
+             person("David","LeBauer", role = c("aut", "cre"),
+                email = "dlebauer@illinois.edu"),
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: @dlebauer is this still the right address to be using here?

**Comment by tonygardella** [comment]
Do we want to add all authors to these lists? As in the authors for every function? I do know there are lot's more authors on this package.

File: modules/data.atmosphere/DESCRIPTION (hunk 1 above)
Comment (mdietze) [reply modules/data.atmosphere/DESCRIPTION]: I'd recommend updating to dlebauer@email.arizona.edu

**Comment by infotroph** [comment]
@tonygardella I was just aiming to get rid of a few lines from the (copious) `R CMD check` output while I was thinking about it, but adding relevant authors is good and worthy. If you have bandwidth to do a more serious edit, I'm happy to leave this PR open for you to push into and merge them all at once.

**Pull request merged**